util/start_test

```diff
@@ -992,6 +992,19 @@ def set_up_executables():
 
     os.environ["NUMLOCALES"] = str(args.num_locales)
 
+    if args.max_locales == "0":
+        logger.write('[max_locales: "(default)"]')
+    else:
+        logger.write('[max_locales: "{0}"]'.format(args.max_locales))
+        os.environ["CHPL_TEST_MAX_LOCALES"] = str(args.max_locales)
```

Comment:
It appears so. The argument types [default to strings](https://docs.python.org/3/library/argparse.html#type) unless specified with `parser.add_argument(type=)`. The casts don't hurt anything, but they aren't necessary.